resources/locations.json

```diff
@@ -0,0 +1,212 @@
+[
+  {
+    "name": "MENSA_ARCISSTR",
+    "long_name": "Mensa Arcisstra\u00dfe",
```

Comment:
Why is it necessary, to have a name and a long name? 
To me it does not seem, as if the name will ever be displayed?

Shouldn't we try to reproduce the same format as in the canteens.json? So the `long_name` should be `name`, and the mensa-id would be needed.

Reply:
Agree. This is a relic from a previous implementation.

Reply:
You are basically right. But `Canteen` inherits from `Enum`. `Enum` however has the field `name` which is the name of the enum, like `MENSA_ARCISSTR` in the above example. In your approach, the `name` attribute in the Json object would mean something completely different than the `name` attribute in the `Canteen` object.  


Instead, I think, one should use the `name` attribute as the id. I only introduced the `canteen_id` now to keep backward compability.

Reply:
Okay alight, however in my opinion the output of the API should be determined by the usefulness and information of the output, and not by the internal mechanics of the programming language used to generate it.
Additionally, it feels wrong to me, to use a field called `name` as id.

Just imagine, we switch to another mechanic, and use some other class/configuration file, etc. to generate the locations we would have to change the API again.

Additionally, if we want to use the new IDs/names (which i think is not necessary by now) we also have to rename all the folders of the current API. However, I think as long as the API format makes sense, we should keep it that way, and not change it due to some internal mechanics.

Reply:
Also making such significant changes will break almost all implementations.

Reply:
You are right about the API. I applied your suggestions now.